sks.go

```diff
@@ -212,6 +212,36 @@ func (c *SKSCluster) ScaleNodepool(ctx context.Context, np *SKSNodepool, nodes i
 	return nil
 }
 
+// EvictNodepoolMembers evicts the specified members (identified by their Compute instance ID) from the
+// SKS cluster Nodepool.
+func (c *SKSCluster) EvictNodepoolMembers(ctx context.Context, np *SKSNodepool, members []string) error {
+	instances := make([]string, len(members))
+
+	for i := range members {
+		id := members[i]
+		instances[i] = id
+	}
```

Comment:
I don't understand the goal of this part
why not use `members` directly?

Reply:
You're right, this isn't necessary – fixed in 30b5016.